src/BugsnagUnity/IDelivery.cs

```diff
@@ -42,7 +44,15 @@ void ProcessQueue()
       {
         try
         {
-          SerializeAndDeliverPayload(Queue.Dequeue());
+          if (Application.internetReachability == NetworkReachability.NotReachable) {
+            Debug.LogWarning("Bugsnag: Network not available, temporarily suspending delivery");
```

Comment:
This log level is unnecessarily high as disconnecting from the network is a common condition. It would be preferable to remove it altogether as it would have the same effect as the current failure log.